_includes/blog-section.html

```diff
@@ -0,0 +1,23 @@
+<!-- blog -->
+<section class="section">
+  <div class="container">
+    <div class="row">
+      <div class="col-12 text-center">
+        <h2 class="section-title">Blogs</h2>
+      </div>
+      {% for post in site.posts offset:0 limit:3 %}
```

Comment:
This is some cool liquid template stuff. :3 :+1:

Reply:
:+1: